readme.md

```diff
@@ -139,18 +109,140 @@ This will create the following files
+    gsutil cp -n "gs://ut-dts-agrc-geocoding-dev-result/*-all_errors_job.csv" ./../data/geocoded-results
+    ```
+
+1. Finally, rebase the results back into the original data with the cli
+
+  ```sh
+  python -m cli post-mortem rebase --single="*-all_errors_job.csv"
+  ```
+
+Now, the original data is updated with this new runs results to fix any hiccups with the original geocode attempt.
+
+#### Second post mortem round
+
+The second post mortem round is to see if we can correct the addresses of the records that do not match using the sweeper project.
+
+1. We need to remove the results of the first round so they do not get processed. Delete the `*-all_errors_job.csv` from the `data/geocoded-results` folder.
+
+1. Post mortem the results to get the current state.
+
+    ```sh
+    python -m cli post-mortem
+    ```
+
+1. Try to fix the unmatched addresses with sweeper.
+
+    ```sh
+    python -m cli post-mortem normalize
+    ```
+
+1. Create a job for the normalized addresses
+
+    ```sh
+    python -m cli create jobs --input-jobs=./../data/postmortem --single=normalized.csv
+    ```
+
+1. Upload the data for the job
+
+    ```sh
+    python -m cli upload --input-folder=./../data/postmortem --single=normalized.csv
+    ```
+
+1. Apply the job in the kubernetes cluster
+
+    ```sh
+    kubectl apply -f ./../jobs/job_normalized.yml
+    ```
+
+1. When that job has completed you can download the results with `gsutil`
+
+    ```sh
+    gsutil cp "gs://ut-dts-agrc-geocoding-dev-result/*-normalized.csv" ./../data/geocoded-results
+    ```
+
+1. Rebase the results back into the original data with the cli
+
+    ```sh
+    python -m cli post-mortem rebase --single="*-normalized.csv" --message="sweeper modified input address from original"
+    ```
+
+1. Finally, remove the normalized csv and run post mortem one last time to get synchronize reality
+
+    ```sh
+    python -m cli post-mortem
+    ```
+
+### Enhance Geodatabase
+
+The geocode results will be enhanced from spatial data. The cli is used to create the gdb for this processing. The layers are defined in `enhance.py` and are copied from the OpenSGID.
+
+1. The geocoded results will need to be renamed to be compatible with a file geodatabase.
+
+    ```sh
+    python -m cli rename
+    ```
+
+1. Create the enhancement geodatabase
+
+    ```sh
+    python -m cli create enhancement-gdb
+    ```
+
+1. Enhance the csv's in the `data\geocoded-results` folder. Depending on the number of enhancement layers, you will end up with a `partition_number_step_number.csv`.
+
+    ```sh
+    python -m cli enhance
+    ```
+
+1. Merge all the data back together into one `data\results\all.csv`
+
+    ```sh
+    python -m cli merge
+    ```
 
-It is recommended to run `all_errors.csv` and `post-mortem` those result. Make sure to update the job to allow for `--ignore-failures` or it will most likely fast fail. Then `post-mortem normalize` the results and run those again to be thorough.
+### Create Deliverable
```

Comment:
Are you missing steps here?

Reply:
sort of. I might just remove this heading for now since pretty much the merge result is what we deliver and it's an email to nathan.